geode-assembly/src/distributedTest/java/org/apache/geode/session/tests/Jetty9CachingClientServerTest.java

```diff
@@ -23,12 +23,14 @@
 
 import javax.servlet.http.HttpSession;
 
+import org.junit.Ignore;
 import org.junit.Test;
 
 import org.apache.geode.cache.Region;
 import org.apache.geode.internal.cache.InternalCache;
 import org.apache.geode.test.dunit.rules.ClusterStartupRule;
 
+@Ignore("GEODE-8704")
```

Comment:
Why are ignoring these tests?

Reply:
These tests are causing pipeline failures following a JDK version update. Since it's only tests in this class, we opted to ignore the tests while we fix the issue to unblock the pipeline.